StudioCore/MapStudioNew.cs

```diff
@@ -590,32 +590,6 @@ private void Update(float deltaseconds)
             {
                 if (ImGui.BeginMenu("File"))
                 {
-                    if (_projectSettings == null || _projectSettings.ProjectName == null)
-                    {
-                        ImGui.MenuItem("No project open", false);
-                    }
-                    else
-                    {
-                        if (ImGui.BeginMenu($@"Settings: {_projectSettings.ProjectName}", Editor.TaskManager.GetLiveThreads().Count == 0))
-                        {
-                            bool useLoose = _projectSettings.UseLooseParams;
-                            if ((_projectSettings.GameType == GameType.DarkSoulsIISOTFS || _projectSettings.GameType == GameType.DarkSoulsIII) && ImGui.Checkbox("Use Loose Params", ref useLoose))
-                            {
-                                _projectSettings.UseLooseParams = useLoose;
-                            }
-                            bool usepartial = _projectSettings.PartialParams;
-                            if (_projectSettings.GameType == GameType.EldenRing && ImGui.Checkbox("Partial Params", ref usepartial))
-                            {
-                                _projectSettings.PartialParams = usepartial;
-                            }
-                            ImGui.EndMenu();
-                        }
-                    }
-
-                    if (ImGui.MenuItem("Enable Texturing (alpha)", "", CFG.Current.EnableTexturing))
```

Comment:
I'd like to keep enable texturing here for now because it's such a common flag for me to toggle to test things

Reply:
Understandable, will do. Would it be acceptable to put them in both menus, and make the File Menu option debug build only?

Reply:
I'd prefer it in the upper menu since it's a pretty big and high visibility option for now

Reply:
Added it back to File menu.